java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -817,4 +817,23 @@ public void functionLoad(boolean withRoute) {
         assertEquals(libName, promise2.get());
         // TODO test with FCALL
     }
+
+    @Test
+    @SneakyThrows
+    public void randomKey() {
+        String key1 = "{key}" + UUID.randomUUID();
+        String key2 = "{key}" + UUID.randomUUID();
+
+        assertEquals(OK, clusterClient.set(key1, "a").get());
+        assertEquals(OK, clusterClient.set(key2, "b").get());
+
```

Comment:
Add tests for cluster with single node route and multi node route.
Add tests for standalone including switching between DBs - empty and nonempty ones.